### ISSUE TITLE
Self-deploy: user interface improvment

### ISSUE THREAD

**Issue by hasop** [fusioninventory/fusioninventory-for-glpi#2114]
Hi,

Here an improvement for the self-deploy interface:
It'll be great if the users are allowed (manually or, better, automatically) to force the agent inventory on their computer when they launch a new task.
Thus, the deployment package can be installed immediately and not later.

Thank you

**Comment by wawax** [comment]
Hi,

Have you check the self service deployment feature ?

**Comment by ddurieux** [comment]
It's the case and works when the browser the user deploy in on same computer where he deploy the package

**Comment by carlrom** [comment]
Hi,
I have a problem with that part too : the deployment works if i force the call to http://127.0.0.1:62354/now, but this call seems blocked by my browsers.
with firefox 56 : no call is done (nothing in devtools/network), although the html source show me the "link rel='import'..."
with internet explorer 11 : same result as firefox
with Chrome 61 : the call to "http://127.0.0.1:62354/now" is explicitely "canceled" by the browser and appears in red in devtools/network

The glpi installation is on https.

**Comment by ddurieux** [comment]
If this call blocked, you need wait the agent run itself
